hadoop-crypto/src/test/java/com/palantir/crypto2/hadoop/EncryptedFileSystemTest.java

```diff
@@ -266,4 +266,67 @@ public void testGetCipherAlgorithm_nonDefault() throws IOException, URISyntaxExc
         assertThat(efs.getCipherAlgorithm(), is(cipherAlg));
     }
 
+    @Test
+    public void testDelete_successful() throws IOException {
+        testDeleteFile(false);
+    }
+
+    @Test
+    public void testDelete_recusiveDeleteOnFile() throws IOException {
+        testDeleteFile(true);
+    }
+
+    @Test
+    public void testDelete_recursiveDeleteOnDir() throws IOException {
+        OutputStream os = efs.create(path);
+        os.write(0x00);
+        os.close();
```

Comment:
move this block into the before method (there's another one to move above in `testRename_successful`